api/admin/routes.py

```diff
@@ -965,6 +1204,94 @@ def sitewide_setting(key):
 @requires_admin
 @requires_csrf_token
 def logging_services():
+    """Manage logging services
+    ---
+    get:
+      tags: 
+        - administration
+      summary: Fetch list of logging services and associated protocols
+      description: Fetch list of logging services and protocols for admins
+      security:
+        - BasicAuth: []
+      responses:
+        200:
+          description: Dict of logging services and associated protocols
+          content:
+            application/json:
+              schema: LoggingServicesGetSchema
+        4XX:
+          description: |
+            An authentication error in which the user could not be authenticated, or 
+            is outherwise un-authorized to perform this action.
+
+            This returns an HTML page with details of the error and a link to the sign-in page.
+          content:
+            text/html:
+              schema: ProblemResponse
+              example: |
+                MISSING_SERVICE
+        5XX:
+          description: |
+            An unanticipated bug in the system that could not be properly handled.
+
+            If the API server is running in debug mode the output will contain a traceback, 
+            otherwise a basic error message will be displayed.
+          content:
+            text/html:
+              example: An internal error occurred
+              schema: ProblemResponse
+    post:
+      tags:
+        - administration
+      summary: Create or update logging services
+      description: |
+        Create or update logging services and associated protocols.
+      security:
+        - BasicAuth: []
+      parameters:
+        - X-CSRF-Token
+      requestBody:
+        required: true
+        content:
+          multipart/form-data:
+            schema: AdminAuthPost
```

Comment:
Is this the same as the `Protocol` object used for the `admin_auth_services` requests? I'm not really familiar with either of these endpoints!

Reply:
These are asking for the same information from the requests in 2 different controller areas. the protocol that is asked for in the form is a string that is used to search protocols by name.